src/main.rs

```diff
@@ -34,6 +34,7 @@ struct QueueConfiguration {
     maps: Vec<String>,
     map_vote_count: u32,
     default_player_data: PlayerData,
+    skill_based_matchmaking: bool, // Indicates if skill-based matchmaking is enabled
```

Comment:
There is a way to remove skill based matchmaking under my system by modifying player config